src/frontend/components/SelectContent/index.tsx

```diff
@@ -421,6 +433,23 @@ export const SelectContent = ({
       </form>
 
       <Tabs>
+        <Tab title="Webinars">
+          <SelectContentSection
+            addMessage={messages.addWebinar}
+            addAndSelectContent={async () => {
+              useCreateVideoMutation.mutate({
+                playlist: playlist!.id,
```

Comment:
Is there any way to remove this assert ?

Reply:
It would be outside of this PR scope, but yes I think we could remove it as this component requires a playlist.